src/map/modifier.h

```diff
@@ -217,9 +217,9 @@ enum class Mod
     SMITE                     = 898, // Raises attack when using H2H or 2H weapons (256 scale)
     TACTICAL_GUARD            = 899, // Tp increase when guarding
 
-    HASTE_MAGIC               = 167, // Haste (and Slow) from magic - 1024 base! (448 cap) Truncate at decimal, do not round.
-    HASTE_ABILITY             = 383, // Haste (and Slow) from abilities - 1024 base! (256 cap?) Truncate at decimal, do not round.
-    HASTE_GEAR                = 384, // Haste (and Slow) from equipment - 1024 base! (256 cap) Truncate at decimal, do not round.
+    HASTE_MAGIC               = 167, // Haste (and Slow) from magic - 10000 base, 375 = 3.75% Truncate at decimal, do not round.
```

Comment:
given that we no longer have to do math and find that we have 40.9% hast and have to clip that to just 40, I think we should remove the bit about rounding/truncation lest we confuse someone into using 300 instead of 375

Reply:
Took care of it myself